lib/themes/dosomething/paraneue_dosomething/scss/patterns/_counter.scss

```diff
@@ -1,12 +1,35 @@
 // @TODO: Move to Forge if this is shipped.
 .counter {
-  display: inline-block;
   color: $med-gray;
   font-weight: $weight-bold;
-  font-size: 13px !important;
-  padding-left: 3px;
+  font-size: 16px;
+  padding-left: 4px;
 
   &.is-active {
-    color: $purple;
+    color: $dark-gray;
+  }
+}
+
+.photo__actions li {
+  * {
```

Comment:
We should avoid using the `*` selector if possible... this could cause unexpected bugs if we ever nested anything else within this container.